### PR TITLE
Create ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by c8y-ui-automations[bot]

Ticket: [MTM-58410](https://cumulocity.atlassian.net/browse/MTM-58410)
Original PR: [6332](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6332)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: Data points graph realtime disabled when tab is in background (#6288) [GRAFT][release/cd] (#6332)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Data points graph realtime updates disabled when browser tab is in background
```

File: content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YdSEScrEC
+    label: Cockpit
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58410
+version: 1019.24.2
+---
+Data points graph realtime disabled when tab is in background (#6288) [GRAFT][release/cd] (#6332)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]: The Claude AI suggested to use the following body:
```suggestion
To improve performance and reduce unnecessary resource consumption, the Cumulocity IoT application now disables realtime updates for data point graphs when the browser tab is in the background or not visible to the user. With this change, data point graphs will only receive and display realtime updates when the browser tab is active and visible. This ensures that system resources are used efficiently without impacting the user experience of viewing data point graphs.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

**Comment by jdreSAG** [comment]
fix release note when https://github.com/SoftwareAG/c8y-docs/pull/2066 is approved

File: content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 1 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:3]:
```suggestion
title: Data points graph realtime is kept enabled when the browser tab is in background
```

File: content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md (hunk 2 above)
Comment (jdreSAG) [inline content/change-logs/application-enablement/ui-c8y-1019-24-2-data-points-graph-realtime-disabled-when-tab-is-in-background-6288.md:17]:
```suggestion
The realtime functionality for the data points graph was automatically disabled when the browser tab was in the background. With this fix, realtime is kept enabled and can be disabled only by direct user interaction.
```

**Review by jdreSAG — APPROVED** [review]

**Comment by jdreSAG** [comment]
same content as https://github.com/SoftwareAG/c8y-docs/pull/2066
fyi @BeateRixen

**Review by sag-lukjan — APPROVED** [review]

**Pull request merged**